src/sql/workbench/contrib/commandLine/electron-browser/commandLine.ts

```diff
@@ -234,7 +239,13 @@ export class CommandLineWorkbenchContribution implements IWorkbenchContribution,
 			}
 
 			const connectionProfile = this.readProfileFromArgs(args);
-			await this._connectionManagementService.showConnectionDialog(undefined, undefined, connectionProfile);
+			await this._connectionManagementService.showConnectionDialog(undefined, {
+				saveTheConnection: true,
+				showDashboard: true,
+				params: undefined,
```

Comment:
sometimes we're including `params = undefined` and other call we don't.  Any reason why?

Reply:
They would mean the same thing, I can remove from all spots to keep it consistent.